### DISCUSSION TITLE
multilingual menu

### DISCUSSION THREAD

**Discussion by HDInfautre** [templaza/astroid-framework#762] [Q&A] [answered]
Hi
I've read the documentation https://docs.astroidframe.work/header/header/site-menu-method
I have created 3 Astroid modules (I would have preferred to do nothing).
I do have the ‘Language selection for new visitors’ module on my browser settings.
The problem is that, unlike cassiopea, I had to select the French menu rather than the default multilingual menu in the ‘Site menu’ header option of astroid.

Despite this, I can't see the English menu
Remember that everything worked correctly with cassiopea.
Regards

**Comment by Chacapamac** [comment, marked as answer]
**Question:**
• “I have created 3 Astroid modules” —>  What type of modules?

If you have 3 languages, you have to create one template style by language and select the corresponding Menu in each template style

**Comment by HDInfautre** [comment]
HI
Ok indeed Astroid like helix need to affect the main menu module (unlike Cassiopea where I didn't have to do anything)
Thanks

**Comment by HDInfautre** [comment]
HI
Ok indeed Astroid like helix need to affect the main menu module (unlike Cassiopea where I didn't have to do anything)

It works well but I anticipated a site in several languages.
But at startup, there is only one site and to avoid indexing the test pages, I disabled the "language change" module and the system plugin - Language filter, as I did with cassiopea on other websites.
With Astroid, I had the multilingual main menu link instead of the French menu link.
For this to work, I have to leave the system plugin enabled - Language filter ... Strange

Thanks
